src/components/TrackCreateModal/TrackCreateModal.tsx

```diff
@@ -1,71 +1,62 @@
 import React from "react";
-import { useSelector } from "react-redux";
-import { RootState } from "../../store";
-import TrackForm from "../TrackForm/TrackForm";
-import { createTrack } from "../../features/tracks/trackSlice";
-import { toast } from "react-toastify";
 import { useAppDispatch } from "../../hooks/redux-hook";
+import { createTrack } from "../../features/tracks/trackSlice";
+import { selectAllTracks } from "../../features/tracks/trackSlice";
+import { selectCreateModalOpen, closeCreateModal } from "../../features/ui/modalSlice";
+import { useTrackQueryParams } from "../../hooks/useTrackQueryParams";
+import TrackForm from "../TrackForm/TrackForm";
 import ToastMessage from "../UI/ToastMessage/ToastMessage";
-import ModalWrapper from "../UI/ModalWrapper/ModalWrapper";
+import { toast } from "react-toastify";
```

Comment:
Nitpick: try to have a structure for you imports. Looks messy right now. Not ronaldo at all.